physical_property/logP/logP_challenge_instructions.md

```diff
@@ -8,8 +8,14 @@ Octanol may be found in the aqueous phase. The mole fraction of water in octanol
 
 - Fill one [`submission_template/logP_prediction_template.csv`](submission_template/logP_prediction_template.csv) template for all molecules predicted with one method. You may submit predictions from multiple methods, but you should fill a separate template file for each different method.
 
+- Your transfer free energy prediction for the neutral form does NOT have to be `microstate000` (the challenge provided microstate). If you use a microstate other than the challenge provided microstate, please use a different `Molecule ID` in the form of `SMXX_extra001` (number can vary). In the `METHOD DESCRIPTION SECTION` in the submission file, please include the `Molecule ID` and SMILES string of the microstate that was used.
```

Comment:
"other than the challenge provided microstate," shoudl be "other than the challenge provided microstates,", right?

Reply:
I would mention here, too -- to avoid people missing it below -- the need to list the microstates used in descending order of population.

Reply:
Updated